.eslintignore

```diff
@@ -0,0 +1 @@
+/dev/new-rule/templates/*
```

Comment:
As the files derived from these templates will be commited to the code base, it might make sense to make eslint pass for them?

Reply:
The derived files should pass but these won't have enough information yet to pass.

Reply:
`.eslintrc.js` could take another override:
```
  overrides: [
    {
      files: ['dev/new-rule/templates/**/*.js'],
      rules: {
        'node/no-missing-require': 'off',
        'no-unused-vars': 'off',
      },
    },
  ],
```

Reply:
I had it like that and I was asked to change it to how I do it now; it seems like it's conflicting opinions at this point and I would prefer to either accept a separate PR to do this or something like that. I don't want to keep going in circles if we can avoid it.

Reply:
Oh, didn’t know that. 👍